### PR TITLE
Added complete Czech language translation.

### PR DESCRIPTION
**Pull request by JimmyS83**
All correction and also spell checks was done. Translation is coherent across my czech translations of Android Telegram, Telegram Web, Plus Messenger and Telegram-FOSS. You can add me to the transifex translate team, so I can update new strings easilly in future. There is some problem @Transifex, because I cant request czech language through Transifex interface. My username there is JimmySmith.
Signed-off-by: Jiri Spicak jiri.spicak@mail.ru (github: JimmyS83)

### CODE REVIEW

File: update.bat [hunk 1]
```diff
@@ -0,0 +1,2 @@
+git fetch upstream
+git pull upstream dev
```
Comment (In-line) [inline update.bat]: Delete this file from commit.

Comment (JimmyS83) [inline update.bat]: Hi, deleted.

**Comment by In-line** [comment]
And read [this](https://github.com/telegramdesktop/tdesktop/blob/master/.github/CONTRIBUTING.md) too

**Comment by JimmyS83** [comment]
I read that. But as I wrote on original PR:
_There is some problem @Transifex, because I cant request czech language through Transifex interface. My username there is JimmySmith. Could you please add me to the translation team there?_ 

I tried to contact to admins on every possible channel, no one answer. I read the translation manual on the web and act accordingly.

I added PR, so maybe some of admin could see that, make a conclusion about quality and maybe fix the Transifex issue for me, so I can procede official steps as it should be. Also other users maybe could benefit from those files @Github meanwhile.

**Comment by john-preston** [comment]
@JimmyS83 The localization approach will be changed in a month or two. For now no more official languages is added, but with the new localization system there will be a way to add all existing languages.

**Comment by JimmyS83** [comment]
@john-preston Could you please confirm, that also in new system I can use existing translation files? I just started with translation Windows Phone strings. So I would like to know I didnt do useless work. Also, I spent many hours by translating Desktop/Web/Android files, so it would be very unfortunate, to start those translations from the scratch in the future..

**Comment by john-preston** [comment]
@JimmyS83 I don't know for sure about the files.

**Comment by yaron-tracxpoint** [comment]
It's me, from the near future, over a month or two, any updates?

**Comment by JimmyS83** [comment]
Yeah, new platform is launched, but Czech language is missing, no place to ask / request :(

**Comment by stek29** [comment]
@JimmyS83 just wait a bit, once everything stabilises new langs will be added :)

**Comment by JimmyS83** [comment]
Hope die last, yes.. But I dont see any official plans or rules to the non-listed languages. Nor the official statement. Which is little bit frustrating considering Czech, which has been translated almost complete (I cant find the 100% list of WP strings, but lets say this is dying platform anyway).

**Comment by stek29** [comment]
@JimmyS83 Well, let's say I was told in private that it will happen :)

**Comment by JimmyS83** [comment]
@stek29 Do you think we (translators) could do anything to attribute that process? We are maintaining transifex's CIS translation, would be much better to continue on official platform..

**Comment by auchri** [comment]
The new translation platform is already here: https://translations.telegram.org

**Comment by stek29** [comment]
@auchri but you can't request or add new languages on it (yet?)

**Comment by JimmyS83** [comment]
@stek29 thats correct AFAIK. I even saw my polish "colleague" (I am taking care of Czech) to try attribute through some comments @ english language there.. Desperation is high! :)

**Comment by JimmyS83** [comment]
Any news regarding adding new language opportunities - officially?
Telegram X was released, which I dont like much (its even [bigger](https://i.imgur.com/mWjjixQ.jpg) than Telegram), but most imporantly, I cant import custom language; or at least, I dont know how.

The risk, that in some time will be even custom language option unavaible is horrifying. Even nowadays I have big difficulty, to persuade other people (even with telegram.zde.cz guide), after that it will be impossible.

I still dont understand, where is the catch.. New platform looks quite fine, language could be released into prod after checking.. :(

**Comment by stek29** [comment]
@JimmyS83 Neither do I, sadly :( I really wish I could help but I simply can.

**Pull request closed without merging**